app/kubernetes/views/deploy/deployController.js

```diff
@@ -41,6 +41,7 @@ class KubernetesDeployController {
       templateId: null,
     };
 
+    this.deployNamespace = '';
```

Comment:
why do you need this on the controller instace? looks like it can be just a regular const, no?